PowerShell/ScubaGear/baselines/defender.md

```diff
@@ -2,45 +2,19 @@
 
```

Comment:
For [policy group 4](https://github.com/cisagov/ScubaGear/blob/1393-update-scbs-with-new-front-matter-text-approved-by-occ/PowerShell/ScubaGear/baselines/defender.md#license-requirements-3), need to add the "However, third party services may be used to meet the DLP requirements." (or similar language as you see fit) to indicate 3rd party service usage.

Reply:
added!